package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "mocha tests/lint-test",
-    "test": "mocha tests --recursive"
+    "test": "mocha tests node-tests --recursive"
```

Comment:
oh, wait, you're just talking about this repo. Yeah, I can do that, though I think it might be worth keeping our tests consistent with what the blueprint-test blueprint generates.